src/shared-components/alert/style.js

```diff
@@ -53,7 +53,7 @@ export const AlertContainer = styled.div`
   justify-content: space-around;
   margin: 0 auto ${SPACING.xsmall};
   opacity: ${props => (props.exiting ? '0' : '1')};
-  padding: ${SPACING.xsmall} 0px;
+  padding: 7px 0px;
```

Comment:
@benkolde i think this is a 1px difference. Do we prefer the spacing constant here?

Reply:
Jk i just read the reason on the trello card. 7px + 1px border = 8px